### PR TITLE
Add Paradox Interactive Icon

### LINKED ISSUE #11510: Request: Paradox Interactive
### Brand Name

Paradox Interactive

### Website

https://www.paradoxinteractive.com/

### Popularity Metric

Similiarweb Global Rank: [43,574](https://www.similarweb.com/website/paradoxinteractive.com)

### Official Resources for Icon and Color

- SVG: https://www.paradoxinteractive.com/favicons/safari-pinned-tab.svg
- Color: `#000000`, from icon

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by lsyversen**
**Issue:** closes #11510

### Popularity Metric

Similiarweb Global Rank: [43,574](https://www.similarweb.com/website/paradoxinteractive.com)

### Checklist

- [X] I updated the JSON data in `_data/simple-icons.json`
- [X] I optimized the icon with SVGO or SVGOMG
- [X] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11339,6 +11339,11 @@
             "hex": "000000",
             "source": "https://docs.paperspace.com/img/ps-logo-light.svg"
         },
+        {
+            "title": "Paradox Interactive",
+            "hex": "000000",
+            "source": "https://www.paradoxinteractive.com/favicons/safari-pinned-tab.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:11345]: I believe we should just link to the homepage here.
```suggestion
            "source": "https://www.paradoxinteractive.com"
```
And also we should make sure your SVG, based on this pinned tab icon, matches https://www.paradoxinteractive.com/images/logo/logo-platypus-white.svg (the header icon on the main site).

File: icons/paradoxinteractive.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....6-15-2.9-17.9-3.5-1.4-.3-6.1-1-10.5-1.7-4.4-.6-8.3-1.2-8.6-1.4-3.1-1.9-32.8-3.2-35.8-1.6-1.9 1-2 1.7-1.6 6.9.3 3.1.9 7.6 1.4 9.9 1.4 7.8 1 44.4-.5 48.7-2.2 6.2-5.7 9.1-10.9 9.1-7.7 0-12.2-6.3-17.2-23.6-9.1-31.8-10.8-36.1-17.7-43.6C322.9 10 317.4 7 312.8 7c-1.8 0-6 1.1-9.3 2.5M210.7 225c10.3 11.1 13.7 33.6 6.2 41.1-2.6 2.6-5.2 2.4-11.6-.7-11.6-5.7-16.7-14.5-14-24.4 1.8-6.9 8.5-19.6 10.9-20.7 2.5-1.1 3.8-.4 8.5 4.7m354.5 11.2c14.4 17.6 25.1 39.7 27.9 57.8 1.1 7.1.8 19.4-.4 20.7-.8.7-23.2.6-31.2-.2-12.3-1.2-21.9-5.7-26-11.9-5.9-8.9-2.3-25 11.7-51.6 11.1-21.4 12-22.1 18-14.8M440.9 344c1.1 4.7 2.3 10.7 2.6 13.5.3 2.7.7 6.8 1 9 .2 2.2.7 7.1 1 11 1.4 16.1 1.8 19.1 5 32 5.1 21 5.4 28.3 1.5 37.2-2.9 6.8-6 9.7-10.4 9.5-9.9-.4-18.7-9.1-18.9-18.6-.1-3.9 1-19.3 1.8-26.6.2-1.9.7-6.9 1.1-11 .6-7.7.7-8.6 1.8-18 .7-5.7 3.3-22.1 4-25.5.3-1.1 1.2-5.2 2.1-9 .9-3.9 1.8-7.8 2.1-8.8 1.5-7.8 2.2-7.1 5.3 5.3m-178.8 10.9c14.9 11.2 23.2 24 23.5 36.2.2 7.6-.2 8.3-5.8 9.3-7 1.3-16.8-4.1-21.3-11.8-3.2-5.4-4.5-12-4.9-24-.3-10.2.1-14.6 1.4-14.6.4 0 3.6 2.2 7.1 4.9M76.7 377c19 8.4 21.2 32.2 4 43.8-13.8 9.3-29 5-37-10.5-1.9-3.7-2.2-12.8-.6-18.6 1.6-6 11.3-15.4 17.4-17 3.7-.9 11.4.2 16.2 2.3" fill="#000000"/></svg>
```
Comment (uncenter) [inline icons/paradoxinteractive.svg:1]: As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon, no other attributes for path elements are allowed.
```suggestion
<svg role="img" viewBox="0 0 700 700" xmlns="http://www.w3.org/2000/svg"><path d="M303.5 9.5c-7.7 3.1-10.5 6.3-10.5 11.8 0 4.1 2.4 14.9 7.3 32.7 3.2 11.8 3.5 25.3.8 34.1-2.3 7.2-3.4 8.2-7.9 7-9.4-2.7-15.9-9.2-30.4-30.1-17.3-24.9-22.5-29.2-36.2-29.6-9.1-.2-11.2.3-21.4 5.3-7.3 3.6-11.2 7.2-11.2 10.2.1 5.2 11.9 22 27.1 38.6 10.7 11.6 18.1 27.5 16.1 34.7-1.5 5.4-5.6 11.1-10.2 14.3-3.8 2.7-5.1 3-11.4 3-11.7-.1-16.6-4.1-36-29-18.5-23.8-26.5-30.1-38.6-29.9-9.6.1-15.9 4.4-18.7 12.9-1.4 4.2-.1 8.8 4.2 15 3.9 5.8 3.9 5.7 18.2 21.1 15.3 16.4 16.8 19.7 16 33.1-.6 10.9-1.6 12.1-8.8 10.5-6.1-1.5-15.8-5.3-29.9-11.9-19.9-9.4-23.1-10.4-34.1-10.7-11-.2-15 .7-20.5 4.9-4.6 3.6-10.1 11.2-10.9 15.2-1.2 6.6 1.2 9.4 13.5 15.8 12.2 6.4 15.5 9 17.6 14.1 3.1 7.3 1.2 16.9-6.7 35.6-8.5 20.2-9.5 26.2-5.9 38.3 3 10.2 3.2 12.4 1.4 18.4-1.9 6.6-4.8 11.2-9 14-5.5 3.8-13.2 3.3-30.3-1.8-9-2.8-10.3-3-16.7-3.8C8.1 301.9-.3 312.2.5 327.6c.5 8.7 1.2 12.4 5.4 28.7 4.6 17.6 4.8 23.5 1.4 35.5-1.6 5.6-2.5 11.5-2.6 16.7-.1 7.2.4 9 4 18 5.2 12.8 5.2 13 6.8 28.5 1 10.4 2.7 17.5 4.9 21.2 5.7 9.5 13.1 10.6 23.5 3.3 4.1-2.8 5.1-3.7 18.2-15.7 13.7-12.4 18.5-15.4 23.3-14.2 5.5 1.3 10.6 10.3 9.9 17.3-.4 4.2-2.2 7.2-6.5 10.5-3.3 2.4-20.1 11.1-30.8 15.9-13.3 5.8-18 10.7-18 18.8 0 10.1 10.9 23.3 33.5 40.9 37.2 28.8 48.9 50.3 43.8 80.5-1.1 6.6-7.6 31-9.8 36.6-.6 1.6 3.6 2.2 14.6 2.2 12.5-.1 31.4-17 40.7-36.6 8.6-17.8 11.1-29 11.3-50.1.1-11.4-.7-15.4-8.1-38.1-10-30.6-13-42.5-13.4-53.3-.4-8.4 0-11.1 5.9-39.2 2.7-12.4 3.8-21.1 3.8-30 .1-9.5-.1-11.1-2.5-16-1.5-3-4.5-7.3-6.8-9.5l-4-3.9-3.1 1.8c-4.4 2.6-10.7 2.2-17.4-1-6.7-3.3-10.4-7.7-10.9-12.8-.5-4.9 3.2-8.8 14.7-15.1 10.8-6 12.8-8.1 13.4-13.8.8-6.5-2.4-12.1-13.9-24.2-13.7-14.6-15.8-17.7-15.3-22.9.8-9.4 10.7-21.5 22.1-26.8 5.3-2.5 7.8-3 13.2-3 12.7.1 29.8 5.6 41.7 13.5 13.2 8.9 15.3 14.5 10.8 29.7-2.5 8.3-2.6 9.1-2.7 14.7-.1 7.6 1.7 9.7 17.3 19.2 8 4.9 9.9 13.1 6 25.6-5.7 18.7-5.2 21.7 5.4 27.8 13.2 7.8 14.8 11 13.3 28.5-.9 12.1 0 17.4 3.8 21.5 2.6 2.8 7.2 4.5 8.6 3.1.4-.5 1.1-5.5 1.4-11.2.6-10.1 1.8-14.6 4.9-18.7 1.5-2 1.6-1.9 3.9.9 1.2 1.7 4.7 6.4 7.5 10.5 8.3 12 9.4 13.3 15.1 18.8 4.9 4.7 10.4 7.4 11.9 5.9 2.8-2.8-3.1-27.6-9.7-40.5-1.2-2.4-2-4.5-1.7-4.7.2-.2 3.3 1.1 6.8 3.1 14.7 7.9 18.5 9.5 22.4 9.5 4.6 0 5.7-.6 7.4-3.7 1.9-3.7 1.3-7.2-3.1-16.2-4.1-8.5-4.2-8.7-4.1-19.6 0-6.1.2-15.3.4-20.5.5-15.9-2.1-18.7-32.7-34.6-8.2-4.2-16.8-9-19.2-10.5-2.4-1.6-4.7-2.9-5.2-2.9s-3.6-2.6-6.9-5.8c-5.1-5-6-6.5-6.7-10.7-.6-4.3-.3-5.6 2.1-10.2 4.8-9.3 9.7-10.8 37.5-11.7 8-.2 16.2-.7 18.4-1 6.8-1.1 14.3-5.5 21.5-12.5 19.1-18.7 22.2-34.6 11.5-59.1-4.9-11.3-5-16.9-.6-26.2 6.8-14.2 21.4-23.9 41.2-27.3 9.8-1.7 28.6-1.3 40.8 1 1.5.3 4.1.7 5.7 1.1 20.9 3.8 39.1 15.8 46.2 30.2 2.2 4.4 2.8 7.3 3 12.9.4 18.1-5.7 22.8-32.7 24.8-11.5.9-15.7 3-24.9 12.2-12.9 13.1-24.1 32.4-24.2 41.9-.1 7.3 1.3 12.3 7.1 25.5 5.3 11.9 6.8 18.4 6.8 29.7 0 9.4-2.2 25.9-4.8 35.2-.7 2.5-1.4 5.4-1.6 6.5s-3.5 12.6-7.3 25.5c-10.5 35.6-10.3 34.9-9.6 46.4.9 14.8-1.1 17.3-16.3 19.9-8.3 1.5-17.1 5.1-20.8 8.5-3.1 2.9-11.1 8.3-20 13.5-9 5.2-13.1 8.8-16.8 14.4-3.4 5.1-4 9-1.7 10.3 2 1 3.4.8 13.5-1.9 3.8-1 7.8-1.6 8.8-1.3 2.5.6 2.3 4.3-.7 13.2-4.7 13.7-6.5 20.9-6.5 25.7 0 5.8 1.7 12.3 3.2 12.3.6 0 5.4-4.2 10.7-9.4 9.1-8.9 14.1-12.5 16.9-12 .6.1 2.3.2 3.7.3 2.9.1 4.7 3.4 9 17.1 2.8 8.8 7.7 19.4 9.5 20.5 1.3.8 4.6-1.1 5.5-3.2.4-1 .8-5.6 1.1-10.3.2-4.7.6-13.4.8-19.3.6-12 2.1-16.5 8.1-24.4 4.3-5.6 6-9.4 7-15.4 1.3-7.5 2.7-9.2 13.2-14.8 5.4-3 14.1-7.8 19.3-10.7 6-3.3 12.8-6.1 18-7.5 7.6-1.9 10.1-2.1 23.8-1.6 8.4.4 15.7.9 16.3 1.2.5.4 1.3-.6 1.9-2 1.4-3.6 0-9.2-11.3-46.5-3.2-10.8-4.7-15.9-5.3-18.7-.1-.6-.7-2.7-1.3-4.5-3.3-10.5-6.5-32.3-6.6-45.3 0-7.7.1-8 2.4-8 1.8 0 3 1.3 5.6 6 1.7 3.3 6.8 11.5 11.1 18.2s10.7 16.8 14.2 22.5c9.2 15 16.6 22.8 21.8 22.8 2.3 0 5.4-3.1 7.2-7.2 1.1-2.6 6.4-29.3 7.7-38.4.8-5.9 5.1-17 8.6-22.6 3.9-6.1 11.5-12.9 17.6-15.9 7.1-3.4 17.9-3.6 25.7-.4 15.2 6.2 27 23.8 30.5 45.1 1 6.1.7 31.2-.4 37.4-3.9 22.4-13.4 44.8-23.7 56.4-4.1 4.7-7 6.5-19.2 12.6-21.9 10.8-23.1 12.1-22.6 26 .8 20.5.5 27.4-1.3 31.9-5.8 14-22.3 29.3-50.8 46.9-21.9 13.5-27.4 18-34.5 28.1-3.8 5.4-8.2 10.3-10.6 11.9-3.9 2.5-4.9 2.7-16.7 3-14.2.3-17 1-25.6 6.2-5.3 3.2-10.4 6.1-21 12-1.6.9-9.5 5.1-17.5 9.4-18 9.6-19.4 10.6-18.6 13.4 2.8 8.7 14.2 13.1 32.7 12.6 11.2-.3 15.9-.9 29.7-4 18.9-4.2 24.1-7.3 37.1-22.4 7.8-8.9 10.7-9.8 23-6.9 2.4.5 7 1.3 10.2 1.6 5.6.5 6.3.3 13.1-3.7 11.5-6.7 46.4-29.6 64.8-42.5 3-2.1 6.8-4.8 8.5-5.9 1.6-1.2 3.4-2.6 4.1-3.1.6-.6 2.6-2.1 4.4-3.5 6.1-4.6 12.2-11.5 14.2-15.9 1.8-3.9 1.9-5.6 1.3-13-.4-4.7-.9-10.1-1-11.9-.4-5.1 1.3-10 4.2-11.9 3.4-2.2 3.9-2.3 15.9-2.7 9.1-.3 10.7-.6 15-3.1 6.3-3.7 19-16.4 24.3-24.3 11.3-16.7 20.9-42 25.1-65.7 2.5-14.3 3.7-18.7 8.9-32.3 2.6-7 5.2-14.1 5.6-15.7 2.5-9.3.6-15.2-7.8-24.3-8.2-8.9-9.1-11-8.4-20.6.3-5.9 1.6-11.5 4.6-20.3 4.5-13.2 5.4-21.6 2.9-28.3-1.7-4.7-3.4-6.8-8.8-11.2-5.4-4.3-9.2-6.2-23-11.8-21.8-8.7-28.1-14.5-28.1-25.4 0-12.5 8.2-15.7 34.6-13.2 5 .5 11.5.9 14.5 1l5.5.1V243c-.1-9.7-.4-12.1-2.4-20-5.6-21.4-16.5-32.4-32.1-32.3-5.9 0-15.8 2.3-18.5 4.3-.5.4-3.9 1.7-7.5 3-14.7 5-26.6-.3-28.6-12.7-1.1-7 .7-10.2 12.3-20.9 18.1-16.7 20.3-19.5 20.3-25.2 0-5-2.1-9-9.3-18-9.3-11.5-10-13.1-10.3-23.6-.2-5-.5-10-.8-11.2-1-3.6-5.1-4.2-20.7-2.9-10.4.9-14.1-.3-24.8-8-4.9-3.6-9.4-6.5-10-6.5-.5 0-4.9 4.8-9.7 10.7-21.5 26.6-34.8 35.4-43.5 29.2-5.4-3.8-6.9-11.6-5.4-27.4.9-9.6.6-27.6-.5-33.1-1.7-8.2-4.1-12.4-10.1-17.8-8.5-7.7-16.3-10.8-35.9-14.5-13.3-2.6-15-2.9-17.9-3.5-1.4-.3-6.1-1-10.5-1.7-4.4-.6-8.3-1.2-8.6-1.4-3.1-1.9-32.8-3.2-35.8-1.6-1.9 1-2 1.7-1.6 6.9.3 3.1.9 7.6 1.4 9.9 1.4 7.8 1 44.4-.5 48.7-2.2 6.2-5.7 9.1-10.9 9.1-7.7 0-12.2-6.3-17.2-23.6-9.1-31.8-10.8-36.1-17.7-43.6C322.9 10 317.4 7 312.8 7c-1.8 0-6 1.1-9.3 2.5M210.7 225c10.3 11.1 13.7 33.6 6.2 41.1-2.6 2.6-5.2 2.4-11.6-.7-11.6-5.7-16.7-14.5-14-24.4 1.8-6.9 8.5-19.6 10.9-20.7 2.5-1.1 3.8-.4 8.5 4.7m354.5 11.2c14.4 17.6 25.1 39.7 27.9 57.8 1.1 7.1.8 19.4-.4 20.7-.8.7-23.2.6-31.2-.2-12.3-1.2-21.9-5.7-26-11.9-5.9-8.9-2.3-25 11.7-51.6 11.1-21.4 12-22.1 18-14.8M440.9 344c1.1 4.7 2.3 10.7 2.6 13.5.3 2.7.7 6.8 1 9 .2 2.2.7 7.1 1 11 1.4 16.1 1.8 19.1 5 32 5.1 21 5.4 28.3 1.5 37.2-2.9 6.8-6 9.7-10.4 9.5-9.9-.4-18.7-9.1-18.9-18.6-.1-3.9 1-19.3 1.8-26.6.2-1.9.7-6.9 1.1-11 .6-7.7.7-8.6 1.8-18 .7-5.7 3.3-22.1 4-25.5.3-1.1 1.2-5.2 2.1-9 .9-3.9 1.8-7.8 2.1-8.8 1.5-7.8 2.2-7.1 5.3 5.3m-178.8 10.9c14.9 11.2 23.2 24 23.5 36.2.2 7.6-.2 8.3-5.8 9.3-7 1.3-16.8-4.1-21.3-11.8-3.2-5.4-4.5-12-4.9-24-.3-10.2.1-14.6 1.4-14.6.4 0 3.6 2.2 7.1 4.9M76.7 377c19 8.4 21.2 32.2 4 43.8-13.8 9.3-29 5-37-10.5-1.9-3.7-2.2-12.8-.6-18.6 1.6-6 11.3-15.4 17.4-17 3.7-.9 11.4.2 16.2 2.3"/></svg>
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
@lsyversen are you still interested in working on this pull request?

**Comment by uncenter** [comment]
Closing as no response from PR author, can reopen if you have still have an interest in working on it! Future contributors- feel free to pick up where this left off in a new PR.

**Comment by ankammaraosandu** [comment]
hello @lsyversen i am new contributer can i take issue and solve it . please help me as this is my first contribution
Thank You

**Comment by uncenter** [comment]
> hello lsyversen i am new contributer can i take issue and solve it . please guide me as this is my first contribution Thank You

Hi, feel free to read through https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md and raise a pull request adding the icon.

**Pull request closed without merging**